### DISCUSSION TITLE
Is there any way to add a new user group?

### DISCUSSION THREAD

**Discussion by jay-jjwu** [frangoteam/FUXA#877] [Q&A]
Hi
question as the tiltle, Is there any way to add a new user group?

**Comment by unocelli** [comment]
Hi, would be possible by modifying the code, there are no settings to add a group

**Reply by jay-jjwu** [reply]
Hi unocelli, thanks for the answer, any guidline to increase the groupe number to 16 or 32 roles?

**Reply by unocelli** [reply]
Hi, you have to check permission attribute in AlarmProperty, NaviItem, GaugeProperty, Script class and where are used the functions:
- UserGroups.GroupsToValue
- UserGroups.ValueToGroups
